src/JackAudioInterface.cpp

```diff
@@ -277,6 +279,14 @@ void JackAudioInterface::jackShutdown(void*)
     // std::exit(1);
 }
 
+//*******************************************************************************
+void JackAudioInterface::jackInfoShutdown(jack_status_t code, const char* reason,
+                                          void* arg)
+{
+    std::cout << reason << std::endl;
+    JackAudioInterface::jackShutdown(arg);
+}
+
```

Comment:
If using the jack_on_info_shutdown() callback fixes the issue then we should replace the old jack_on_shutdown() callback rather than adding a second callback. Per the JACK API documentation:

NOTE: if a client calls this AND jack_on_shutdown, then in the event of a client thread shutdown, the callback passed to this function will be called, and the one passed to jack_on_shutdown() will not.